modules/oadp-deleting-backup-repository.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-deleting-backups-doc.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-deleting-backup-repository_{context}"]
+= Deleting a backup repository
+
+After you delete the backup, and after the Kopia repository maintenance cycles to delete the related artifacts, are complete, there is no metadata, or manifest objects referencing the backup. You can then delete the `backuprepository` custom resource (CR) to complete the backup deletion process.
+
+.Prerequisites
+
+* You have deleted the backup of your application.
+* You have waited up to 72 hours after the backup is deleted. This time frame allows Kopia to run the repository maintenance cycles.
+
+.Procedure
+
+. To get the backup repository CR for a backup, run the following command:
++
+[source,terminal]
+----
+$ oc get backuprepositories.velero.io -n openshift-adp
+----
+
+. To delete the backup repository CR, run the following command:
++
+[source,terminal]
+----
+$ oc delete backuprepository <backup_repository_name> -n openshift-adp # <1>
```

Comment:
I believe the `#` is only necessary for YAML blocks ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-asciidoc-callouts-in-the-yaml)):
```suggestion
$ oc delete backuprepository <backup_repository_name> -n openshift-adp <1>
```